docs/content/latest/api/ycql/dml_insert.md

```diff
@@ -68,6 +68,7 @@ Where
 - **NOTE**: You should either use the `USING TIMESTAMP` clause in all of your statements or none of
   them. Using a mix of statements where some have `USING TIMESTAMP` and others do not will lead to
   very confusing results.
+- Inserting rows with TTL is not supported on tables that have indexes. 
```

Comment:
@ddorian --

on tables that have indexes
-->
on tables with transactions enabled.

And maybe add a pointer to what a transaction enabled table in YCQL is.

Reply:
Shouldn't same restriction appear for UPDATE also?

And is there already a github issue to make sure the restriction is enforced in code also?

Reply:
Made the changes.
This is the issue tracking in code: https://github.com/yugabyte/yugabyte-db/issues/6325

Reply:
@kmuthukk can you recheck the latest commit which fixes the notes above ?